libs/ui/toggle/brain/src/lib/brn-toggle.directive.ts

```diff
@@ -39,15 +38,13 @@ export class BrnToggleDirective {
 	public id = `brn-toggle-${BrnToggleDirective.uniqueId++}`;
 
 	@Input()
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
-	public value: any;
+	public value: string | number | null = null;
```

Comment:
Should probably also allow undefined, as Signals default to undefined (as opposed to the async pipe which defaults to null) - there will definitely come a follow up ticket about this.

Reply:
I think this should remain any. It gives developers the most flexibility and we shouldn't dictate what they use as their value. It's also how the Angular CDK works and I wanna keep that